example_school_library_decorator/classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
```

Comment:
- [ ] The students `getter and setter` is also missing here make sure you add it as you initialize in the constructor. Kindly refer to this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor`)